athena-vertica/src/main/java/com/amazonaws/connectors/athena/vertica/VerticaJdbcSplitQueryBuilder.java

```diff
@@ -0,0 +1,297 @@
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.connectors.athena.vertica;
+
+import com.amazonaws.athena.connector.lambda.domain.predicate.Constraints;
+import com.amazonaws.athena.connector.lambda.domain.predicate.Range;
+import com.amazonaws.athena.connector.lambda.domain.predicate.SortedRangeSet;
+import com.amazonaws.athena.connector.lambda.domain.predicate.ValueSet;
+import com.google.common.base.Joiner;
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import com.google.common.collect.Iterables;
+import org.apache.arrow.vector.types.Types;
+import org.apache.arrow.vector.types.pojo.ArrowType;
+import org.apache.arrow.vector.types.pojo.Field;
+import org.apache.arrow.vector.types.pojo.Schema;
+import org.apache.commons.lang3.Validate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.stringtemplate.v4.ST;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.*;
+import java.util.stream.Collectors;
+/**
+ * Query builder for database table split.
+ */
+public class VerticaJdbcSplitQueryBuilder {
+    private static final Logger LOGGER = LoggerFactory.getLogger(VerticaJdbcSplitQueryBuilder.class);
+    private static final String EXPORT_BUCKET = "export_bucket";
+    private final String quoteCharacters;
+
+    /**
+     * @param quoteCharacters database quote character for enclosing identifiers.
+     */
+      public VerticaJdbcSplitQueryBuilder(String quoteCharacters) {
+        this.quoteCharacters = Validate.notBlank(quoteCharacters, "quoteCharacters must not be blank");
+    }
+
+    /**
+     * Common logic to build Split SQL including constraints translated in where clause.
+     *
+     * @param jdbcConnection JDBC connection. See {@link Connection}.
+     * @param catalog        Athena provided catalog name.
+     * @param schema         table schema name.
+     * @param table          table name.
+     * @param tableSchema    table schema (column and type information).
+     * @param constraints    constraints passed by Athena to push down.
+     * @param split          table split.
+     * @return prepated statement with SQL. See {@link PreparedStatement}.
+     * @throws SQLException JDBC database exception.
+     */
+    public String buildSql(
+            final String schema,
+            final String table,
+            final Schema tableSchema,
+            final Constraints constraints,
+            final String queryID)
+            throws SQLException {
+
+        //get the bucket where export results wll be uploaded
+        String s3ExportBucket = System.getenv(EXPORT_BUCKET);
+        StringBuilder sql = new StringBuilder();
+
+        String exportSqL = "EXPORT TO PARQUET(" +
+                "directory = 's3://"+ s3ExportBucket + "/"+queryID.replace("-","")+ "'" +
+                ", Compression='snappy'\n" +
+                ", fileSizeMB=1000) \n" +
+                "AS \n";
```

Comment:
String template will be in a future version? I see you have the dependency in the POM but was surprised to see this string concat.